job_definitions/export_data.yml

```diff
@@ -0,0 +1,35 @@
+{% set environments = ['preview', 'production'] %}
```

Comment:
Mostly to make it easier to test. It doesn't upload files anywhere, but you can still check the CSVs from the workspace directory.